packages/react-positioning/src/createTargetFromClick.ts

```diff
@@ -0,0 +1,30 @@
+import * as PopperJs from '@popperjs/core';
+
+export type PopperVirtualElement = PopperJs.VirtualElement;
+
+/**
+ * Creates a virtual target for popper based on the position of a click event
+ * Useful for scenarios such as context menus
+ */
+export function createTargetFromClick(nativeEvent: MouseEvent): PopperVirtualElement {
```

Comment:
```suggestion
export function createVirtualElementFromClick(nativeEvent: MouseEvent): PopperVirtualElement {
```

nit: I suggest different naming there (according to typings and description)